konduit-serving-models/konduit-serving-tensorflow-python/src/main/java/ai/konduit/serving/models/tensorflowpython/KerasStep.java

```diff
@@ -0,0 +1,131 @@
+/* ******************************************************************************
+ * Copyright (c) 2020 Konduit K.K.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+
+package ai.konduit.serving.models.tensorflowpython;
+
+import ai.konduit.serving.annotation.json.JsonName;
+import ai.konduit.serving.annotation.runner.CanRun;
+import ai.konduit.serving.pipeline.api.context.Context;
+import ai.konduit.serving.pipeline.api.data.Data;
+import ai.konduit.serving.pipeline.api.data.ValueType;
+import ai.konduit.serving.pipeline.api.step.PipelineStep;
+import ai.konduit.serving.pipeline.api.step.PipelineStepRunner;
+import ai.konduit.serving.pipeline.api.step.PipelineStepRunnerFactory;
+import ai.konduit.serving.pipeline.util.DataUtils;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+import org.nd4j.common.base.Preconditions;
+import org.nd4j.python4j.PythonGIL;
+import org.nd4j.shade.jackson.annotation.JsonProperty;
+
+
+@lombok.Data
+@Accessors(fluent = true)
+@JsonName("KERAS")
+@NoArgsConstructor
+public class KerasStep implements PipelineStep {
+    private String modelUri;
+    private String[] inputKeys;
+    private String[] outputKeys;
```

Comment:
We'll need 2 things here:
(a) javadoc on the class and fields
(b) `@Schema(description = "...")` on the class and the fields